frontend/lib/map/preview/accepting_store_preview.dart

```diff
@@ -52,7 +52,7 @@ class AcceptingStorePreview extends StatelessWidget {
     return AcceptingStoreSummaryModel(
         item.id,
         item.store.name,
-        item.store.description ?? "Keine Beschreibung verfübar",
```

Comment:
Why did you remove this again? (Also there was a typo in "verfübar")

Reply:
Was unnecessary since the frontend/lib/store_widgets/accepting_store_summary.dart widget is used in both places.

Reply:
Ah, I see 👍🏽